src/main/java/de/agilecoders/wicket/requirejs/AmdModuleHeaderItem.java

```diff
@@ -19,78 +19,77 @@
  */
 public class AmdModuleHeaderItem extends JavaScriptHeaderItem implements IReferenceHeaderItem {
 
-    /**
-     * Creates a {@link AmdModuleHeaderItem} for the given reference.
-     *
-     * @param reference resource reference pointing to the javascript resource
-     * @return A newly created {@link AmdModuleHeaderItem} for the given reference.
-     */
-    public static AmdModuleHeaderItem forReference(JavaScriptResourceReference reference) {
-        return forReference(reference, reference.getName());
-    }
+	/**
+	 * Creates a {@link AmdModuleHeaderItem} for the given reference.
+	 *
+	 * @param reference resource reference pointing to the javascript resource
+	 * @return A newly created {@link AmdModuleHeaderItem} for the given reference.
+	 */
+	public static AmdModuleHeaderItem forReference(JavaScriptResourceReference reference) {
+		return forReference(reference, reference.getName());
+	}
```

Comment:
This just changes the indentation. There are no meaningful changes.
Please revert the change.